libs/vkd3d/device.c

```diff
@@ -6949,6 +6946,15 @@ static void d3d12_device_caps_init_feature_options11(struct d3d12_device *device
             device->device_info.properties2.properties.limits.minStorageBufferOffsetAlignment <= 16;
 }
 
+static void d3d12_device_caps_init_feature_options13(struct d3d12_device *device)
+{
+    D3D12_FEATURE_DATA_D3D12_OPTIONS13 *options13 = &device->d3d12_caps.options13;
+
+    /* Negative viewports become core in Vulkan 1.2. */
+    assert(VKD3D_MIN_API_VERSION >= VK_API_VERSION_1_2);
```

Comment:
This assert is misleading. We always required negative viewports. Non-negative viewports in D3D12 become negative viewports in Vulkan. Negative viewport in D3D12 will become non-negative I think.

Reply:
Right right. Removed assert.